### ISSUE TITLE
Branch guidance in the readme is unclear

### ISSUE THREAD

**Issue by fake-name** [snuf/iomemory-vsl#40]
From `README.md`:

> **Only the master branch is tested, nothing else**. It is currently running in a CEPH cluster with official LTS kernel 4.14 and works quite well. Testing and running with the kernel module has been done on Ubuntu and Redhat. *** the untested branch has only been compiled and loaded not device tested ***

> At the moment there is a lot of code redesign ongoing. You can find the bleeding edge development in the next_generation branch. **Despite its name it is more mature than the master.** Especially it fixes several known bugs:

(emphasis mine)

Only `master` is tested, but `next_generation` is more mature? And there'a mention of a `untested` branch that does not seem to exist at all. Additionally, there are a *bunch* of miscellaneous branches that appear to be ancient and unused.

------

> Installing the fio-util, fio-common, fio-preinstall and fio-sysvinit are recommended. When moving from source 2.3.11 to source 3.2.10 the firmware HAS TO BE UPDATED to 3.2.10.20150212. For the move to 3.2.15 from 3.2.10 I've not seen any issues with my card.

When going from `2.3.11 to source 3.2.10` (*down* grading?), you need to update the firmware?

I think this is saying that if you want to use a firmware more recent then 3.2.10, you need to update the card firmware to at least 3.2.10?

------

Basically, the readme needs to be clarified, and it'd probably also be nice to have the ancient branches removed so they don't clutter the things up.

**Comment by plappermaul** [comment]
Good point. Should be better now.

**Comment by fake-name** [comment]
Much better! 

The "Other Notes" section is still a bit unclear, but other then that I think you've nailed it.
